package.json

```diff
@@ -516,12 +516,12 @@
                 {
                     "command": "appService.Refresh",
                     "when": "view == azureAppService && viewItem == subFolder",
-                    "group": "1_folderGeneralCommands@2"
+                    "group": "1_folderGeneralCommands@1"
```

Comment:
Not sure about this

Reply:
`subFolder` is any folder below the "Files" node. For example "HttpTriggerJS" in my case below:
![screen shot 2018-09-06 at 9 25 45 am](https://user-images.githubusercontent.com/11282622/45171317-ea6d5c80-b1b6-11e8-9457-a2aa81fe033d.png)

I believe "subFolderRefreshCommand" might be more accurate. But honestly the numbers are the only part you _really_ need. The "_subFolderRefreshCommand" part is just for our own readability and I find it more annoying to maintain than helpful. In the the functions repo, I don't use that part at all: https://github.com/Microsoft/vscode-azurefunctions/blob/master/package.json#L373

In other words, I don't really care what put as long as it works :) Please post one screenshot for each case that you changed